### ISSUE TITLE
Can we please add the option to work with a tokenized dataset, escpailly for the CPT task. 

### ISSUE THREAD

**Issue by shamanez** [huggingface/alignment-handbook#144]
Since we have the CPT task now, it would be nice to have the ability to feel a tokenized and packed dataset directly.

**Comment by BramVanroy** [comment]
This is actually supported by TRL (if "input_ids" is in the dataset no additional packaging or preprocessing happens), but not in the scripts in the alignment-handbook. You can implement it yourself though:

- add input_ids, labels, attention_mask to `columns_to_keep` when calling `get_datasets`
- remove the section where the logger is printing some random samples, or wrap it in an if-clause, e.g. `if train_dataset and "input_ids" not in train_dataset.column_names:`

From the top of my head, that should suffice to get it to work.
